### PR TITLE
Man pages: refactor common options

### PR DESCRIPTION
**Pull request by edsantiago**
Continued. Harder-to-review ones this time.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
man pages: continued refactoring and cleanup of common options
```

### CODE REVIEW

**Comment by edsantiago** [comment]
At this point there is simply no sane way that I can think of to review this. There's a lot of human judgment involved here, mixing and matching and picking what I think is the best way to phrase things.

File: hack/markdown-preprocess [hunk 1]
```diff
@@ -29,7 +29,7 @@ def process(infile):
     # Some options are the same between containers and pods; determine
     # which description to use from the name of the source man page.
     pod_or_container = 'container'
-    if '-pod-' in infile:
+    if '-pod-' in infile or '-kube-' in infile:
```
Comment (edsantiago) [inline hack/markdown-preprocess:32]: The `kube` man pages tend to speak in terms of pods, so I choose to treat them as pod man pages for purposes of `<<container|pod>>` expansion.

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**